src/FakeItEasy/ExceptionMessages.cs

```diff
@@ -63,6 +63,12 @@ internal static class ExceptionMessages
         public static string CannotRaiseEventWhenCallingBaseMethod =>
             "The fake cannot raise the event because event subscription calls the base implementation.";
 
+        public static string HandleEventsOnStrictFakes =>
+            @"Strict fakes don't handle calls to event accessors unless configured to do so. You can either:
+- configure the calls explicitly;
+- create the fake with the StrictFakeOptions.ManageEvents flag;
+- or use Manage.Event(eventName).Of(fake) or Manage.AllEvents.Of(fake) to enable automatic event management.";

```

Comment:
Super nit (from the future): should we use `nameof`s or the like to build the message so that we'll be forced to rewrite this if the calls move or change? I am perhaps worrying too much.